docker-entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# bundle install --jobs 20 --retry 5
+
+rails db:create
+rails db:migrate
+rails db:seed
```

Comment:
i would be cautious about doing db seeding and migration in the entry point as this could cause issues when scaling or container restarts doing database commands that arent expected (annoying in dev disastrous in live) maybe consider doing the db work as another service in your compose file? 

something like
db_seed:
        build: .
        command: bundle exec rails db:create db:migrate db:seed
        volumes:
            - .:/jinglejam
        depends_on:
            - db
        environment:
            - FROM_EMAIL_ADDRESS=jingle-jam@example.com
            - HMAC_SECRET=c360bcae-df63-4616-808a-860f03d7da6b

Reply:
This has been changed so that it occurs in the docker_install.sh (Im not sure this is the right way to do it), please correct me.